.github/workflows/build-and-generate.yml

```diff
@@ -0,0 +1,59 @@
+name: Build & Generate project files
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16, 20]
```

Comment:
maybe just use 20? 16 is pretty old

Reply:
Yeah, I put 16 just to check the compatibility of the requirement in the package.json file `"node": ">=16"`. I agree that we should focus on more recent versions of node.js and maybe change the requirement in package.json file?

Reply:
Yes, i noticed that in the package.json but forgot to mention that we should probably support node >=v20